src/Microsoft.ServiceFabric.Actors/Generator/ActorNameFormat.cs

```diff
@@ -310,6 +310,22 @@ public static string GetLocalEseStoreConfigSectionName(Type actorImplementationT
                 GetActorServiceName(actorImplementationType));
         }
 
+        /// <summary>
+        /// Gets Key Value store configuration section name specified in configuration package for the actor service.
+        /// </summary>
+        /// <param name="actorImplementationType">Type of class implementing the actor.</param>
+        /// <returns>Key Value store configuration section name.</returns>
+        /// <remarks>Values specified in Key Value store configuration section are used to configure <see cref="System.Fabric.KeyValueStoreReplicaSettings"/>
+        /// for storing the state of actor.
+        /// </remarks>
+        public static string GetKeyValueStoreSettingsConfigSectionName(Type actorImplementationType)

```

Comment:
some unit tests and `sample app with new config section` will help validate the scenario.